src/cirrus/release.py

```diff
@@ -195,6 +216,9 @@ def new_release(opts):
     config.update_package_version(new_version)
     changes = ['cirrus.conf']
 
+    if requirements_updated:
+        changes.append('requirements.txt')
```

Comment:
I could, but I am trying to be paranoid with other errors that could happen since its been set.  If people are fine with just checking opts.bump I can go with that.